lightly/transforms/mae_transform.py

```diff
@@ -59,10 +44,11 @@ def __call__(self, image: Union[Tensor, Image]) -> Tensor:
         Applies the transforms to the input image.
 
         Args:
-            Image (Tensor): The input image to apply the transforms to.
+            image: 
+                The input image to apply the transforms to.
 
         Returns:
-            Image (Tensor): The transformed image.
+            The transformed image.
 
         """
-        return self.transform(image)
+        return [self.transform(image)]
```

Comment:
We should just return the single image and not inherit from `MultiViewTransform` as MAE doesn't really create multiple views.